.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 dist
 yarn-error.log
 coverage
+.vscode
```

Comment:
`nit` add newline

Reply:
I will make this change when I have time.